src/components/organisms/Table/Table.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import HeaderCell from '../../atoms/HeaderCell/HeaderCell';
+
+import { userData } from '../../../data/userData';
+import NameCell from '../../atoms/NameCell/NameCell';
+import PlainTextCell from '../../atoms/PlainTextCell/PlainTextCell';
+import DateCell from '../../atoms/DateCell/DateCell';
+import BooleanCell from '../../atoms/BooleanCell/BooleanCell';
+
+import './Table.css'
+
+export interface OneOrMoreArray<T> extends Array<T> {
+    0: T
+}
```

Comment:
This is interesting, TIL 🙂

Reply:
Ive seen this pattern used most often for tuples but I like being able to insist on arrays of any specified length